pkg/init/Dockerfile

```diff
@@ -1,3 +1,9 @@
+FROM alpine:edge AS build
```

Comment:
this should use `linuxkit/alpine:<latest hash>` just like the next `FROM` line. the alpine base image should have all the tools you need to compile it.